viam_cartographer.go

```diff
@@ -551,6 +556,10 @@ func (cartoSvc *CartographerService) PointCloudMap(ctx context.Context) (func()
 		return nil, ErrClosed
 	}
 
+	if cartoSvc.existingMap != "" && !cartoSvc.enableMapping && cartoSvc.postprocessedPointCloud != nil && cartoSvc.postprocessed.Load() {
```

Comment:
`cartoSvc.existingMap != "" && !cartoSvc.enableMapping` to check if we are in localization mode. 
`cartoSvc.postprocessedPointCloud != nil` to check that the pointcloud has been set.
`cartoSvc.postprocessed.Load()` to check if postprocessed has not been toggled off.

Reply:
Add a comment above this line with this information